validator_worker/src/sentry_interface.rs

```diff
@@ -378,3 +403,151 @@ pub mod campaigns {
         client.get(endpoint).send().await?.json().await
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use adapter::DummyAdapter;
+    use primitives::{
+        adapter::DummyAdapterOptions,
+        config::{configuration, Environment},
+        sentry::Pagination,
+        util::tests::{
+            discard_logger,
+            prep_db::{
+                ADDRESSES, DUMMY_CAMPAIGN, DUMMY_VALIDATOR_FOLLOWER, DUMMY_VALIDATOR_LEADER, IDS,
+            },
+        },
+        UnifiedNum,
+    };
+    use std::str::FromStr;
+    use wiremock::{
+        matchers::{method, path, query_param},
+        Mock, MockServer, ResponseTemplate,
+    };
+
+    #[tokio::test]
+    async fn test_get_all_spenders() {
```

Comment:
Create a Hashmap with all spenders and for each page response create a simple `vec![].collect()` so that you can `assert_eq!` more easily without repetition